ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -511,7 +512,13 @@ + (void)updateViewController:(UIViewController *)vc
                                                                              action:nil];
   [backBarButtonItem setMenuHidden:config.disableBackButtonMenu];
 
-  auto isBackButtonCustomized = !isBackTitleBlank || config.disableBackButtonMenu;
+  auto overrideBackButtonItem =
+      config.disableBackButtonMenu; // when config.disableBackButtonMenu is true we need to set backButtomItem
+  prevItem.backButtonTitle = resolvedBackTitle;
```

Comment:
Hmm, now we do always set `backButtonTitle` of `prevItem` 🤔 We need to test it thoroughly for regressions. We need to prepare list of tests examples to test before we can process further.

Reply:
I reverted that change, seems like it could have negative impact on back button menu.